pyant/phased_paracyl_plots.py

```diff
@@ -0,0 +1,64 @@
+
```

Comment:
This file should __not__ be in the "pyant" directory as it is not a vital library component that performs a function, rather it can be in the "examples" folder and it can be worded like "this illustrates differences between a phased and a non phased finite cylidrical paraboloid reflector radar"

Reply:
Obviously.  I *did* write _work in progress_ in the commit log though.

Reply:
Haha yes :+1: i just wanted to try to do a proper PR review